Functions/xASL_quant_SliceTiming.m

```diff
@@ -111,6 +112,43 @@
 	SliceTiming = (0:1:(nSlices-1)) * x.Q.SliceReadoutTime;
 elseif length(x.Q.SliceReadoutTime) == nSlices
 	SliceTiming = x.Q.SliceReadoutTime;
+	
+	% Check for SliceTiming consistency between slices if a vector was provided
+	if nSlices > 1
+		SliceTimingDiff = SliceTiming(2:end) - SliceTiming(1:(end-1));
+		% See if the slice-timing difference between slices is equal for all slices
+		[SliceTimingDiffUnique,SliceTimingDiffIndex] = uniquetol(SliceTimingDiff,0.01);
+		
+		% Make sure that the positive values are listed first
+		[SliceTimingDiffUnique,SliceTimingDiffIndexSort] = sort(SliceTimingDiffUnique,'descend');
+		SliceTimingDiffIndex = SliceTimingDiffIndex(SliceTimingDiffIndexSort);
+		
+		if length(SliceTimingDiffUnique) > 2
+			fprintf('Warning: Inconsistent SliceTiming between slices\n');
+		elseif length(SliceTimingDiffUnique) == 2
+			% This could be multi-band
+			bIsMB = false;
```

Comment:
would write out `MultiBand` instead of `MB` to keep code readable

Reply:
Good comment. Changed on all occurences.